bofh@runner2:~/rpmbuild/SOURCES$ diff -up linux-3.10.0-693.2.2.el7-org/arch/x86/kernel/smpboot.c linux-3.10.0-693.2.2.el7/arch/x86/kernel/smpboot.c
--- linux-3.10.0-693.2.2.el7-org/arch/x86/kernel/smpboot.c      2017-10-03 10:44:19.978064057 +0200
+++ linux-3.10.0-693.2.2.el7/arch/x86/kernel/smpboot.c  2017-10-03 11:02:38.182306317 +0200
@@ -338,6 +338,7 @@ static void __init smp_init_package_map(
         */
        max_physical_pkg_id = DIV_ROUND_UP(MAX_LOCAL_APIC, ncpus);
 
+    #ifdef CONFIG_XEN
     if (x86_hyper == &x86_hyper_xen_hvm) {
         /*
         * RHEL-only. Each logical package has not more than
@@ -348,6 +349,7 @@ static void __init smp_init_package_map(
         */
         __max_logical_packages = min(max_physical_pkg_id, total_cpus);
     }
+    #endif /* CONFIG_XEN */
 
        size = max_physical_pkg_id * sizeof(unsigned int);
        physical_to_logical_pkg = kmalloc(size, GFP_KERNEL);
